setup.py

```diff
@@ -40,9 +49,12 @@
 
 if __name__=="__main__":
 
-    if ('upload' in sys.argv) or ('sdist' in sys.argv):
-        import param, numbergen
-        param.__version__.verify(setup_args['version'])
-        numbergen.__version__.verify(setup_args['version'])
+# TODO: conda packages won't have meaningful version number yet
+
+# TODO: need to decide what to do about this
+#    if ('upload' in sys.argv) or ('sdist' in sys.argv):
```

Comment:
You can condition this on the presence of an environment variable when you need this check disabled.

Reply:
That's what my earlier comment says ;) It's hard to follow because this same discussion was happening in parambokeh too. I'll try to keep it here, and I'll ping you to review when I actually try something. Thanks!

(I've never quite understood this line, because aside from sdist there's also various bdist args one might pass instead. And nowadays I'm not sure anyone would do 'python setup.py upload' - I will check that. So some of the delay is that I need to update myself on python packaging.)